src/pages/_proxied-dot-io/packer/index.jsx

```diff
@@ -0,0 +1,199 @@
+import PackerIoLayout from 'layouts/_proxied-dot-io/packer'
+import ProductFeaturesList from '@hashicorp/react-product-features-list'
+import AnimatedTerminal from 'components/_proxied-dot-io/packer/animated-terminal'
+import BrandedCta from 'components/_proxied-dot-io/packer/branded-cta'
+import HomepageHero from 'components/_proxied-dot-io/packer/homepage-hero'
+import IntegrationsTextSplit from 'components/_proxied-dot-io/packer/integrations-text-split'
+import SectionBreakCta from 'components/_proxied-dot-io/packer/section-break-cta'
+
+import s from './home/style.module.css'
+
+function HomePage() {
```

Comment:
It looks like we're missing the background image for the homepage hero:

Actual: https://dev-portal-git-zsadd-packer-hashicorp.vercel.app/

Expected:
<img width="1036" alt="Screen Shot 2022-02-23 at 1 38 41 PM" src="https://user-images.githubusercontent.com/1289701/155395092-c276dd8d-c374-42a4-8643-8bcb58dd42df.png">